fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -85,7 +85,7 @@ object ItemKicker {
 
   def seriesOrBlogKicker(item: Content) =
     item.tags.find({ tag =>
-      Set("series", "blog").contains(tag.`type`) && !TagsThatDoNotAutoKicker.contains(tag.id)
+      Set(TagType.Series: TagType, TagType.Blog).contains(tag.`type`) && !TagsThatDoNotAutoKicker.contains(tag.id)
```

Comment:
Weird typing with the types... it looks like it's mixing types. These are the ones from the thrift definitions I believe?
I wonder what would happen if you filled out the type of the set;
`Set[TagType](TagType.Series, TagType.Blog)`